e2e/__init__.py

```diff
@@ -0,0 +1,4 @@
+import sys
+import os
+
+sys.path.append(os.path.join(__file__))
```

Comment:
Is the `e2e` package's `__init__.py` the right place to set this up? Shouldn't it rather be done in the module(s) (i.e. regular python file(s)) that relies/rely on this? (So that they'll work in isolation, too, without having to import the package that they're part of themselves.)